### DISCUSSION TITLE
Image optimisation

### DISCUSSION THREAD

**Discussion by 1stClassSABR** [vercel/community#7380] [Help]
### Summary

So, we use google vision to to read text from pictures uploaded from users on our app, it has usually taken on average 2-3 seconds on each reading, but suddenly it's taking longer than 10s, which means it stops working? 

We have used this months without any issue, so im just wondering what is causing it?

![undefined - Imgur](https://github.com/vercel/community/assets/70705101/65d4223d-b901-499b-a29a-bce3db073c62)
![image (6)](https://github.com/vercel/community/assets/70705101/0c880e85-541d-471f-9b17-1c2b1cfe3a37)

### Steps to Reproduce

Not sure, I will ask my developer.

**Comment by 1stClassSABR** [comment]
update*

we were able to change the max timeout to 60, and it works, eventually, but it takes so long, in past it took 2 to 3 seconds

**Reply by pawlean** [reply]
Hi @1stClassSABR!

Hmm - that's odd. Did you change anything to the website / app codebase?

[a comment by an automated account is not shown]
